1.9/administration/installing/cloud/azure.md

```diff
@@ -4,17 +4,15 @@ nav_title: Azure
 menu_order: 1
 ---
 
-This document explains how to install DC/OS by using the Azure Marketplace.
-
-You can alternatively skip the marketplace and install directly using the <a href="https://downloads.dcos.io/dcos/stable/azure.html" target="_blank">DC/OS templates for Azure Resource Manager</a>.
+This document explains how to install DC/OS 1.9 through the Azure Marketplace.
```

Comment:
Let's remove this doc from 1.9 for now (there aren't any templates for 1.9 yet).